Description: no CPP macro timestamps
 Reporducible builds project support, see: 
 http://wiki.debian.org/ReproducibleBuilds/TimestampsFromCPPMacros
Author: Milan Kupcevic <milan@debian.org>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/main.c
+++ b/main.c
@@ -679,10 +679,10 @@ int main(int argc, char * argv [])
    * Print out an identifying string so folks can tell what version
    * they are running
    */
-  avrdude_message(MSG_NOTICE, "\n%s: Version %s, compiled on %s at %s\n"
+  avrdude_message(MSG_NOTICE, "\n%s: Version %s\n"
                     "%sCopyright (c) 2000-2005 Brian Dean, http://www.bdmicro.com/\n"
                     "%sCopyright (c) 2007-2014 Joerg Wunsch\n\n",
-                    progname, version, __DATE__, __TIME__, progbuf, progbuf);
+                    progname, version, progbuf, progbuf);
   avrdude_message(MSG_NOTICE, "%sSystem wide configuration file is \"%s\"\n",
             progbuf, sys_config);
 
